modus/app-manifest.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: App Manifest
 description: "Define the resources for your app"
-"og:title": "App Manifest - Modus"
+"og:title": "App  - Modus"
```

Comment:
What's your thinking behind this change? "App" is pretty broad

Reply:
Strange, I didn't change that. I'll fix this, looks like I accidentally did something.